### PR TITLE
Add performance monitoring to shieldtest

### PR DESCRIPTION
**Pull request by ZeLonewolf**
This PR adds a new column to shieldtest.html that lists the performance of each shield.  This will allow us to assess whether we introduce any performance changes when modifying shield code.

Example:

image

### CODE REVIEW

File: src/shieldtest.js [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 import { map } from "./americana.js";
 import * as shield from "./js/shield.js";
-import { shields } from "./js/shield_defs.js";
```
Comment (ZeLonewolf) [inline src/shieldtest.js:5]: This was just an unused import.

File: src/shieldtest.js [hunk 2]
```diff
@@ -246,13 +245,25 @@ for (let network of networks) {
...
   }
+  let perfEntries = performance.getEntriesByName(`${network}`);
+  var perfDuration = 0;
+  for (let perf of perfEntries) {
+    perfDuration += perf.duration;
+  }
+  let shieldRate = Math.round((1000 * perfEntries.length) / perfDuration);
+  row
+    .insertCell()
+    .appendChild(document.createTextNode(`${shieldRate} shields/sec`));
```
Comment (jleedev) [inline src/shieldtest.js:268]:
```suggestion
    .append(`${shieldRate} shields/sec`);
```

Comment (ZeLonewolf) [reply src/shieldtest.js:268]: That reports in console:
```
Uncaught TypeError: Failed to execute 'appendChild' on 'Node': parameter 1 is not of type 'Node'.
    at shieldtest.js:268:6
```

**Review by jleedev — APPROVED** [review]
Nice.

I should learn the performance mark/entries stuff instead of manually using performance.now() :)

**Pull request merged**